docs/man_pages/bodhi-push.rst

```diff
@@ -64,3 +64,5 @@ If you find bugs in bodhi (or in the mage page), please feel free to file a bug
 request::
 
     https://github.com/fedora-infra/bodhi
+
+Want to know about bodhi `read the documentation <https://bodhi.fedoraproject.org/docs>`_.
```

Comment:
I would probably write this more simply:

```
Bodhi's documentation is available online: https://bodhi.fedoraproject.org/docs
```

You could also change the heading of this section from "Bugs" to "Help" so that it makes more sense to link to the documentation within it (since documentation isn't just for when bugs happen).